### PR TITLE
Add Hindi Font

### LINKED ISSUE #62: Adding some other languages
**Is your feature request related to a problem? Please describe.**
{
     I think it will be more helpful if we can provide support in other languages specially hindi.
     It will help a lot of students !
}
**Describe the solution you'd like**
{
      Well in too short I want to help my younger brother who has hindi as subject. 
      Its just an extension of languages you provide sir. More languages mean more help 
      as your application will have than a much broader audience.
}
**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Himanshunitrr**
Resolves #62

### CODE REVIEW

**Comment by Himanshunitrr** [comment]
how to add that in value="" in index.html ?

**Comment by saurabhdaware** [comment]
In css you can add,
```css
@font-face {
  font-family: 'Hindi Font';
  src: url(fonts/Hindi_Type.ttf);
}
```

and in index.html add value="'Hindi Font'"

**Comment by Himanshunitrr** [comment]
Did that

**Comment by Himanshunitrr** [comment]
But not working. I do not know why? Trying to debug

**Comment by Himanshunitrr** [comment]
The path URL was messed. Finally debugged it.

File: .theia/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.autoSave": "on"
```
Comment (saurabhdaware) [inline .theia/settings.json]: Can you check what file this is? I think it got added automatically

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
Perfect! Can you just remove the `.theia` file, I think it was added by the editor.

File: .theia/settings.json (hunk 1 above)
Comment (Himanshunitrr) [reply .theia/settings.json]: Done

**Review by saurabhdaware — APPROVED** [review]
Awesome! Thank you so much, Himanshu 🎉

**Pull request merged**